build.cake

```diff
@@ -233,7 +259,7 @@ Task("PackageReleaseZipFile")
         var stagingLicenseFile = stagingDirectory.CombineWithFilePath("license.txt");
         CopyFile(licenseFile, stagingLicenseFile);
 
-        var releaseNotesFileName = "Release Notes v" + gitVersion.LegacySemVer + ".md";
+        var releaseNotesFileName = "Release Notes v" + gitVersion.MajorMinorPatch + ".md";
```

Comment:
This small change is needed for the build script to work on a branch other than master.